framework/test/unit/modules/dpos_v2/module.spec.ts

```diff
@@ -1128,6 +1126,9 @@ describe('DPoS module', () => {
 				const currentTimestamp = 10290;
 				const lastForgedHeight = 926;
 				const nextForgedHeight = lastForgedHeight + 1;
+				// Last block slot is not included because it is currently generated
+				// and is not related to this test case
+				const delegatesExceptGenerator = delegateAddresses.slice(0, 102);
```

Comment:
wouldn't it better if we check generator has 0 consecutiveMissedBlocks?

Reply:
It was originally checking, I thought you didn't want it. Setting `consecutiveMissedBlocks` to 0 is a part of the unit since it's in the `_updateProductivity` as a simple assignment, so I think yes it would be good to check it to be sure about `_updateProductivity` unit's functioning as a whole. Do you approve it?